wiki/Beatmaps/Packs/en.md

```diff
@@ -0,0 +1,48 @@
+# Beatmaps Packs
+
+_Main page: [Beatmaps](/wiki/Beatmaps)_
+
+Beatmaps packs, as the title suggests, are a `.zip` files containing beatmaps.
+The name of the pack is based on what they contain (e.g. `Approved Beatmap Pack #7` would only contain [approved](#approved) beatmaps).
```

Comment:
whoops, this was carried over from the Beatmaps page (the page that originally had this)